toolset/databases/postgres/postgres.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:15-bullseye
+FROM postgres:16-bullseye
```

Comment:
I suggest switching to Debian Bookworm as well, to benefit from the work that has gone into C library, i.e. `FROM postgres:16-bookworm`.